schema.xml

```diff
@@ -45,7 +45,7 @@
     that avoids logging every request
 -->
 
-<schema name="Blacklight Demo Index" version="1.5">
+<schema name="Temple Library Search" version="1.5">
```

Comment:
You know, I hadn't noticed that version param there.  Wonder if we should start using it.